hoomd/box.py

```diff
@@ -32,24 +30,24 @@ class (e.g. Scalar3, Int3).
         except (ValueError, TypeError):
             raise ValueError("Expected values of type {}.".format(scalar_type))
     else:
-        raise ValueError("Expected three or one value received {}.".format(
-            len(vec)))
+        raise ValueError("Expected a sequence of three values or a single "
+                         "value. Received {} values.".format(len(vec)))
 
 
-_make_scalar3 = partial(_make_vec_three,
+_make_scalar3 = partial(_make_vec3,
```

Comment:
I think renaming `_make_vec_three` to `_make_vec3` aligned better with how `_make_scalar3` and related functions are used.